internal/store/database_users.go

```diff
@@ -55,3 +60,13 @@ func (s *Store) DeleteDatabaseUser(groupID, username string) error {
 		return fmt.Errorf("unsupported service: %s", s.service)
 	}
 }
+
+func (s *Store) ProjectDatabaseUser(projectID string, opts *atlas.ListOptions) ([]atlas.DatabaseUser, error) {
```

Comment:
[nit] no need for `Project` here on the name, similar to a previous comment for listing methods let's try to make them plural